integration/index.js

```diff
@@ -0,0 +1,31 @@
+const axios = require("axios");
+const axiosRetry = require("axios-retry");
+const { SkynetClient, DEFAULT_SKYNET_PORTAL_URL } = require("../index");
+
+// retry if we're getting rate limited
```

Comment:
Copied from skynet-js. Figured, why not?

Reply:
hello m-cat,

if it works.
I also used the test from "skynet-js" where the
"axios-mock-adapter" is used.